index.html

```diff
@@ -46,69 +52,84 @@ <h2 class="tonic">Tonic</h2>
                    <li><button class="languages-List" type="submit">css</button></li>
                     <li><button class="languages-List" type="submit">javascript</button></li>
                 </ul>
+            </div>
                 <button class="btn-Project" type="submit">See Project</button>
+            </div> 
+            </div>
 
-            </div> <div class="works">
-                <div>
+            <div class="works">
+                <div class="desktop-Img-Works">
                     <img class="snapshoot" src="images/Snapshoot-Portfolio3.svg" alt="Snapshoot-Image">
                 </div>
+                <div class="desktop-Text-Works">
                 <h2 class="tonic">Tonic</h2>
                 <ul class="sub-Points">
                     <li class="canopy">CANOPY</li>
                     <li class="sub-Text">Back End Dev</li>
                     <li class="sub-Text">2015</li>
                 </ul>
                 <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="lang"> 
                 <ul class="languages">
                     <li><button class="languages-List" type="submit">html</button></li>
                    <li><button class="languages-List" type="submit">css</button></li>
                     <li><button class="languages-List" type="submit">javascript</button></li>
                 </ul>
+                </div>
                 <button class="btn-Project" type="submit">See Project</button>
+            </div>
+            </div>
 
-            </div> 
-            <div class="works">
-                <div>
+            <div class="works-Change">
+                <div class="desktop-Img-Works">
                     <img class="snapshoot" src="images/Snapshoot-Portfolio4.svg" alt="Snapshoot-Image">
                 </div>
+                <div class="desktop-Text-Works">
                 <h2 class="tonic">Multi-Post Stories</h2>
                 <ul class="sub-Points">
                     <li class="canopy">CANOPY</li>
                     <li class="sub-Text">Back End Dev</li>
                     <li class="sub-Text">2015</li>
                 </ul>
                 <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="lang"> 
                 <ul class="languages">
                     <li><button class="languages-List" type="submit">html</button></li>
                    <li><button class="languages-List" type="submit">css</button></li>
                     <li><button class="languages-List" type="submit">javascript</button></li>
                 </ul>
+                </div>
                 <button class="btn-Project" type="submit">See Project</button>
             </div>
+            </div>
             <div class="about-Myself">
```

Comment:
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![bad](https://user-images.githubusercontent.com/69638013/154628143-d4bfce06-0408-44aa-853f-815d98ba8a5d.PNG)  | ![good](https://user-images.githubusercontent.com/69638013/154628127-5c18324b-b41e-4aeb-a8aa-8b3343c9be95.PNG)



- [ ] Kindly align this div to the right so it looks like the original design, check the screenshot above 👍🏿 
- [ ] You can reduce the margin left to 3rem 👍🏿

Reply:
completed.